### PR TITLE
Gracefully handle broken or missing models at initial load time

### PR DESCRIPTION
**Pull request by lstein**
# Gracefully handle broken or missing models at initial load time

- If initial model fails to load, invoke.py will inform the user that something is wrong with models.yaml or the models themselves and drop user into `configure_invokeai.py` to repair the problem.
- The model caching system will longer try to reload the current model if there is none. This will get rid of the unsightly spectacle of the system trying to load model "None"
- This also fixes a regression in the listing of models introduced in PR #1525 and not caught when I did the code review.
- Tested and verified with the following scenarios:
   - missing models.yaml file
   - bad line in models.yaml
   - weights line does not point to a proper models file
   - various supporting models (e.g. `clip-vit-large-patch14`) are missing or screwed up

Note that after opening this branch I had to merge in recent changes to `development` in order to get things to run at all, and so there are some extraneous commits showing up in the commit log, as well as the new notebook file. This shouldn't cause any merge issues.

### CODE REVIEW

File: notebooks/Stable-Diffusion-local-Windows.ipynb [hunk 1]
```diff
@@ -0,0 +1,222 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Easy-peasy Windows install"
```
Comment (hipsterusername) [inline notebooks/Stable-Diffusion-local-Windows.ipynb:7]: Is this bit related to the Model handling? This seems to be outside the scope of PR?

**Review by hipsterusername — CHANGES_REQUESTED** [review]

File: notebooks/Stable-Diffusion-local-Windows.ipynb (hunk 1 above)
Comment (hipsterusername) [reply notebooks/Stable-Diffusion-local-Windows.ipynb:7]: (comment pertains to the entire Windows.ipynb)

Comment (lstein) [reply notebooks/Stable-Diffusion-local-Windows.ipynb]: I've removed this file from the PR.

**Review by hipsterusername — APPROVED** [review]
